.travis.yml

```diff
@@ -4,29 +4,38 @@ addons:
       - ubuntu-toolchain-r-test
     packages:
       - libstdc++-4.9-dev
+
 branches:
   only:
     - gh-pages
+
 language: generic
```

Comment:
Still not sure about this choice. Seems wrong, but I haven't looked into it more.